packages/ui-kit/src/components/TimeSeries/utils.ts

```diff
@@ -255,12 +255,18 @@ export function getScales(options: GetScalesOptions) {
       ...scalesBase.x,
       type: 'timeseries',
       time: {
+        isoWeekday: true,
         unit: getGranularityBasedUnit(granularity)
+      },
+      adapters: {
+        date: {
+          zone: options.zone
```

Comment:
From the comment above ^
```suggestion
          zone
```